snakefile

```diff
@@ -20,6 +20,13 @@ rule extract_adm0_gadm:
     shell:
         "curl {params} -o {output} -O -J -L"
 
+rule extract_adm1_gadm:
```

Comment:
Since the GADM Yemen geopackage contains all 3 admin level regions, maybe we can just use a single extraction command, but rename it so that it's clear that all three levels are being extracted

Reply:
I thought about that but maintained a the 1:1 relationship of the de-duplication of extraction effort (if you see what I mean). Something to discuss as a potential principle though

Reply:
See review of same topic